scripts/email.sh

```diff
@@ -0,0 +1,9 @@
+docker image pull docker.isc.ru.nl/rdr/tool/rdr-emailer:latest
```

Comment:
For this script to be useful for others, at least the following should be documented:

- that the script exists and can be used, in the readme and also on our confluence 
- how/when where will the rdr-email docker image be built and pushed to docker.isc.ru.nl (the last run in jenkin is older then the last commit, is it a manual job or broken?)
- that the script expects a recipients.csv and template.txt file
- that the scrip does not require a from address, nor username, password (how does that work? which from adress will it use? Or do I need to uncomment?)

I don't think it's worth the effort of maintaining a  (automated) build for this project. The readme explains how to use it nicely. So if I had to use it, I'd build the docker image on my pc (because I don't have go installed) and run it as described. I'd say we delete the rdr-emailer job in Jenkins (assuming that is what I think it is). Less is more.

Reply:
* I added it to the readme. At this moment I don't think we need a seperate confluence item for it.
* The build job did not poll SCM. I added that, and it works now.
* I also mentioned the recipients.csv and template.txt files
* The from address is the Donders one by default (no-reply@donders.ru.nl, refer to main.go), but could be changed for RDR. The username, password is not required because authentication is based on IP address whitelisting.

I think automating the project is a very small effort to formalize the build procedure, and get an example at the same time. The desired situation would in my opinion be to completely separate the Docker build from running it so we don't have any actual code on the target environments. I know we're not there yet.